src/main/java/com/redhat/devtools/intellij/tektoncd/actions/AddTriggerAction.java

```diff
@@ -81,13 +82,16 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
                 }
 
                 String kind = (element instanceof PipelineNode) ? "Pipeline " : "Task ";
+                telemetry.property(TelemetryService.PROP_RESOURCE_KIND, kind);
                 AddTriggerWizard addTriggerWizard = openTriggerBindingWizard(anActionEvent, element, triggerBindingTemplates, model, kind);
                 if (!addTriggerWizard.isOK()) {
+                    telemetry.result("wizard aborted")
```

Comment:
Shouldn't we have a constant to report abort because I think it's not specific to this action

Reply:
can add a constant but it is imho of limited use. The value is reported as value for the "result" property. Can go for "aborted" instead?